css/sponsors.css

```diff
@@ -160,3 +172,6 @@ h1 {
     font-weight: 600;
     color: #666;
 }
+h1.col-md-12{
+    padding-bottom: 50px;
+}
```

Comment:
This is to give more space between the grids and the title. I think this change makes the page look less cluttered